nemo/collections/llm/recipes/llama31_405b.py

```diff
@@ -279,20 +279,20 @@ def finetune_recipe(
         This recipe uses the SQuAD dataset for fine-tuning. Be aware that fine-tuning a 405B model
         requires substantial computational resources.
     """
-    recipe = default_finetune_recipe(
-        model(), "meta-llama/Meta-Llama-3.1-405B", dir, name, num_nodes, num_gpus_per_node
-    )
+    recipe = default_finetune_recipe(model(), "meta-llama/Llama-3.1-405B", dir, name, num_nodes, num_gpus_per_node)
```

Comment:
This should be consistent with HF model name- https://huggingface.co/meta-llama/Llama-3.1-405B?